docs/src/pages/component-demos/tabs/ScrollableTabsButtonAuto.js

```diff
@@ -0,0 +1,75 @@
+// @flow weak
+/* eslint-disable react/no-multi-comp */
+
+import React, { Component, PropTypes } from 'react';
```

Comment:
Could you rebase the branch on `next` we started the migration to use the `prop-types` packages.